### ISSUE TITLE
Camera asking for permission in Safari(ios) repeatedly whereas permission has been blocked

### ISSUE THREAD

**Issue by mohitrawat2486** [mebjas/html5-qrcode#277]
Camera asking for permission in _safari_ browser so every times whereas previously it has been blocked, working fine in Android chrome browser

**Comment by mohitrawat2486** [comment]
found the solution

**Comment by ROBERT-MCDOWELL** [comment]
would be nice you describe the solution you found for others thanks

**Comment by mohitrawat2486** [comment]
permission issues

**Comment by vincentckk** [comment]
Just tested the following browser, keep asking Permission after Scan, GET/POST redirect to same page.
Firefox
Samsung mobile browser

No problem for the following broswers.
Chrome, Chrome mobile, Edge, Edge mobile

**Comment by ROBERT-MCDOWELL** [comment]
So it's a browser issue, you should contact samsung for that....

**Comment by vincentckk** [comment]
> So it's a browser issue, you should contact samsung for that....

Correct. Also contact Firefox. I also don't thing it is rely on this app.
The browser name is just for reference.
